KitePrintSDK/src/main/java/ly/kite/print/UnitOfLength.java

```diff
@@ -0,0 +1,82 @@
+/*****************************************************
+ *
+ * UnitOfLength.java
+ *
+ *
+ * Modified MIT License
+ *
+ * Copyright (c) 2010-2015 Kite Tech Ltd. https://www.kite.ly
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The software MAY ONLY be used with the Kite Tech Ltd platform and MAY NOT be modified
+ * to be used with any competitor platforms. This means the software MAY NOT be modified 
+ * to place orders with any competitors to Kite Tech Ltd, all orders MUST go through the
+ * Kite Tech Ltd platform servers. 
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ *****************************************************/
+
+///// Package Declaration /////
+
+package ly.kite.print;
+
+
+///// Import(s) /////
+
+
+///// Class Declaration /////
+
+import android.content.Context;
+
+import ly.kite.R;
+
+/*****************************************************
+ *
+ * This class defines units of length.
+ *
+ *****************************************************/
+public enum UnitOfLength
+  {
+  CENTIMETERS ( R.string.unit_centimeters, R.string.unit_centimeters_short ),
+  INCHES      ( R.string.unit_inches,      R.string.unit_inches_short      ),
+  PIXELS      ( R.string.unit_pixels,      R.string.unit_pixels_short      );
+
+
+  private int  mPluralResourceId;
```

Comment:
Same for these, could be final but the enums are effectively immutable anyway due to encapsulation -- I won't keep going on about this unless you wish me to.